### PR TITLE
Dockerfile for ROSE

### PR DESCRIPTION
**Pull request by AnnaDinaburgVulikh**
Hi @sleviim @sfishbai , 
This pr is trying to follow the notes for #309 and raised the issue #427 .

One way of solving the issue is creating a requirements_dev.txt with the needed dependencies.
The question is, would you prefer to lock all the module versions or only `idna`(the conflicting one)?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Use official CentOS Python 2.7 image
```
Comment (sleviim) [inline Dockerfile]: @rollandf , shoulds we use python 3.6 for it?

Comment (rollandf) [reply Dockerfile]: Python 2.7 is not supported, so yes.

Comment (AnnaDinaburgVulikh) [reply Dockerfile]: Hi, i'm not sure where you see this line.
In the files I commited the dicker uses python 3.7

Comment (sleviim) [reply Dockerfile]: Maybe after rebasing it will be removed.
You can see another reference at line 3.

File: README.md [hunk 2]
```diff
@@ -94,18 +94,32 @@ For running the same track for all drivers (instead or random) start the server
...
+
 ## Running a driver
 
 In a new window, open your virtual environment:
 
     pipenv shell
 
-Create your driver file: 
-    
+Create your driver file:
```
Comment (sleviim) [inline README.md:117]: redundent space attheend of the line

**Review by sleviim — CHANGES_REQUESTED** [review]

**Review by sleviim — APPROVED** [review]

**Pull request merged**